comp-layout/wishlist.html

```diff
@@ -0,0 +1,121 @@
+
+
+    
+          <div class="row wishlist">
+               <h2>MY WISHLIST  <i class="fa fa-heart heart"></i></h2>
+          </div>
+          <div class="wishlist-container">
+          <div class="row">
+              <div class="male-section">
+                   <img src="/comp-images/shoe2.jpg"><i class="fa fa-window-close  close" aria-hidden="true"></i>
+                   <h4>WHITE MEN SHOE</h4>
+                   <div class="rating">
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star-o"></i>
+                        <i class="fa fa-heart heart"></i>
+                   </div>
+                   <P class="price">$150.00</P>
+                   <a class="buy-btn" href="#"><small>BUY NOW</small> </a>
+                   <a class="buy-btn" href="#"><small>ADD TO <i class="fa fa-shopping-cart"></i></small> </a>
+              </div>
+
+              <div class="male-section">
+                   <img src="/comp-images/watch1.jpeg"><i class="fa fa-window-close  close" aria-hidden="true"></i>
+                   <h4>CLASSIC WATCH</h4>
+                   <div class="rating">
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star-o"></i>
+                        <i class="fa fa-heart heart"></i>
+                   </div>
+                   <P  class="price">$150.00</P>
+                   <a class="buy-btn" href="#"><small>BUY NOW</small> </a>
+                   <a class="buy-btn" href="#"><small>ADD TO <i class="fa fa-shopping-cart"></i></small> </a>
+              </div>
+
+
+             <div class="male-section">
+                  <img src="/comp-images/blue tshirt.jpg"><i class="fa fa-window-close  close" aria-hidden="true"></i>
+                  <h4>BLUE PLAIN T-SHIRT</h4>
+                  <div class="rating">
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star-o"></i>
+                       <i class="fa fa-heart heart"></i>
+                  </div>
+                  <P  class="price">$150.00</P>
+                  <a class="buy-btn" href="#"><small>BUY NOW</small> </a>
+                  <a class="buy-btn" href="#"><small>ADD TO <i class="fa fa-shopping-cart"></i></small> </a>
+             </div>
+         </div>
+
+
+</div>
+      
+ <!--- footer-->
+
+
+ <div class="foot">
+     <footer class="footer">
+          <div class="footer-container">
+               <div class="webmaker">MADE BY SAURAV DARJI</div>
```

Comment:
Check if this div is serving any purpose.

Reply:
done